docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,103 @@
+
+## Repair Agent
+ecChronos is an agent design to simplify and automate repairs. Each instance of ecChronos is designed to keep a single Cassandra node repaired.
+It first calculates all the ranges that need to be repaired per table and then groups nodes by the ranges they have in common.
+
+One repair is run against a Cassandra node per range as if running repair with -st and -et flags.
+
+A lock table within Cassandra ensures that nodes do not run repairs at the same time. How the locking mechanism behaves
+can be configured in [ecc.yaml](application/src/main/resources/ecc.yml).
+
+ecChronos uses repair history saved in Cassandra to know what is and isn't repaired. It ensures that the same thing
+will not need to be repaired multiple times per interval by keeping track of the latest repaired time. If a repair fails,
+it will retry the range.
+
+Priority is given to every range that needs to be repaired. The priority is based on the number of hours since it has
+been last repaired to ensure that the oldest ranges are repaired first. Failing repairs will therefore gain higher and higher
+priority with time.
+
+## Configuration
+
+The default settings assume a GC Grace Second setting of 10 days. Repair will be run once on every table every interval which
+defaults to 7 days. If 8 days have passed on a range without a repair then a warning will be logged. If 10 days have passed an ERROR is logged.
+
+Tune these alarm parameters to fit your use case in [ecc.yaml](application/src/main/resources/ecc.yml)
+
+## Starting ecChronos
+
+Installation information can be found in [SETUP.md](docs/SETUP.md).
+
+When ecChronos is started, it will assume that a table is repaired if no repair history exists.
+In this case, it will wait the interval and then begin repairing each table.
+
+If more fine grained control is required
+over which tables and keyspaces will be repaired, this can be configured in [schedule.yaml](application/src/main/resources/schedule.yml)
+
+If repair history exists, ecChronos will take the last time repair was run and add the interval for the next repair time.
+
+
+## Ecctool
+
+
+
+### Schedules
+
+Running ecctool schedules will give an overview of the current schedules keeping tables updated.
+
+
+<pre>----------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
instead of html tags i would prefer markdown.
i.e change \<pre\> to "```"

Reply:
Tried that at first but had some issues. Turns out that it was my IDE that was showing strange output.